src/analysis/static/graph/CFG.ts

```diff
@@ -50,4 +51,70 @@ export class CFG {
       .filter((node) => node.type === NodeType.Root)
       .map((node) => <RootNode>node);
   }
+
+  getNodeById(nodeId : string): Node {
+    const found = this._nodes.filter((node: Node) => node.id == nodeId);
+    if(found.length != 1) {
```

Comment:
If you use array.find you can simply use if(found) to check if found is non null.
I also suggest to rename found -> node.